### PR TITLE
Added better Groupassigner

### PR DESCRIPTION
**Pull request by toster234**
I am sending my **better version Group Assigner**.
Assigner has a login through the code that gets in a poke. 
**It is not possible** to change groups of another person.
Assigner is looks nice.

### Configuration

1. Go to /config/assignerconfig.php
2. In line 5 you can change name of rank group
3. In line 6 you can change limit ranks from group
4. In line 7 you add id group

### CODE REVIEW

**Comment by derrobin154** [comment]
Your version is better than mine, but I programmed my groupassigner within 5 minutes :-D

**Comment by toster234** [comment]
Miss click....

**Review by Wruczek — APPROVED** [review]
Would be great to move CSS and JS to separate files

**Comment by derrobin154** [comment]
Can i make a German Language File in this Commit, or i must create a own commit?

**Comment by Wruczek** [comment]
Not sure if this is possible, but try pushing to toster's repository to be included in this commit

**Comment by toster234** [comment]
> Can i make a German Language File in this Commit, or i must create a own commit?

Make PR in my repository

**Review by derrobin154 — CHANGES_REQUESTED** [review]
The Language URL is not found, the Language Code is not en_EN, the Code is en_US or en_GB

**Review by derrobin154 — APPROVED** [review]

**Review by derrobin154 — CHANGES_REQUESTED** [review]
Sorry, en_GB don't exist in this File.. you must take en_US

**Comment by toster234** [comment]
> Sorry, en_GB don't exist in this File.. you must take en_US

My bad i didn't test it.

**Review by derrobin154 — APPROVED** [review]

File: assigner.php [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+require_once __DIR__ . "/config/assignerconfig.php";
+
+$clientgroups = [];
+
+try {
+    $tsAdmin = getTeamspeakConnection("#no_query_clients");
+
+    $userip = $_SERVER["HTTP_CF_CONNECTING_IP"]; // $_SERVER["REMOTE_ADDR"];
+
+    echo "<!-- IP: {$_SERVER["REMOTE_ADDR"]}, CFIP: {$_SERVER["HTTP_CF_CONNECTING_IP"]} -->";
```
Comment (Wruczek) [inline assigner.php:14]: The switch for real IP / proxy IP needs to not be hardcoded, rather added to the assigner config

**Comment by jakub96124124** [comment]
Uncaught TypeError: $(...).selectpicker is not a function
    at assigner.php:277


Please help thanks

**Comment by toster234** [comment]
> Uncaught TypeError: $(...).selectpicker is not a function
>at assigner.php:277
>
>Please help thanks

You didn't replace all files. You can download my fork of ts-website https://github.com/toster234/ts-website

**Comment by jakub96124124** [comment]
image

**Comment by toster234** [comment]
You can copy and paste link XD

**Comment by jakub96124124** [comment]
image

**Comment by jakub96124124** [comment]
Uncaught TypeError: $(...).selectpicker is not a function
at assigner.php:277

**Comment by jakub96124124** [comment]
Uncaught TypeError: $(...).selectpicker is not a function
at assigner.php:277

No work

**Comment by Wruczek** [comment]
2.0 ships with a group assigner, closing.

**Pull request closed without merging**